tests/main/refresh-amend/task.yaml

```diff
@@ -1,22 +1,28 @@
 summary: Ensure that refresh --amend works
 
 restore: |
-    rm -f test-snapd-tools_*.snap
+    rm -f test-snapd-only-in-edge_*.snap
 
 execute: |
     echo "When installing a local snap"
-    snap download test-snapd-tools
-    snap install --dangerous ./test-snapd-tools_*.snap
-    snap list |MATCH "test-snapd-tools.*x1"
+    snap download --edge test-snapd-only-in-edge
+    snap install --dangerous ./test-snapd-only-in-edge_*.snap
+    snap list |MATCH "test-snapd-only-in-edge.*x1"
 
     echo "A normal refresh will not refresh it to the store rev"
-    if snap refresh test-snapd-tools 2> stderr.out; then
+    if snap refresh test-snapd-only-in-edge 2> stderr.out; then
```

Comment:
I hope that snap is not `grade: stable`, so that it cannot be put into stable ever.